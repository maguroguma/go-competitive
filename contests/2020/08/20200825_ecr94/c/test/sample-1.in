3
101110
2
01
1
110
1
