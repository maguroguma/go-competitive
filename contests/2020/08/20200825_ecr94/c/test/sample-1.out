111011
10
-1
